### ISSUE TITLE
Update aspnetapp samples to use bootstrap >= 3.4.1

### ISSUE THREAD

**Issue by MichaelSimons** [Microsoft/dotnet-framework-docker#230]
This is to address a vulnerability in the current version (3.0.0) being referenced.

**Comment by mthalman** [comment]
This requires more work than just updating the package reference to 3.4.1.  Attempting to do that will result in a package conflict with the another referenced package, AspNet.ScriptManager.bootstrap which requires a version of bootstrap that matches its own version.  However, there's not a 3.4.1 version of AspNet.ScriptManager.bootstrap.  So the only way to resolve the vulnerability is to upgrade all the way to 4.3.1 of bootstrap for which there is a matching version of AspNet.ScriptManager.bootstrap.  But there are [major breaking changes](https://getbootstrap.com/docs/4.0/migration/) between v3 and v4 of bootstrap that require updating the app implementation.

Since this project is based off the template defined by VS, I looked at how the template has been changed to account for this.  It looks like VS 2017 hasn't yet been updated.  VS 2019 Preview has been updated to use 3.4.1 but it works around the AspNet.ScriptManager.bootstrap package conflict by shipping its own local copy of a 3.4.1 version of AspNet.ScriptManager.bootstrap package.  The problem is that this is a local package not available on nuget.org so it won't work for our purposes here.
